File: Challenges/Coding World Cup/2023/Day 30 - NED vs AFG - Double Score Beauty.java
import java.util.Arrays;

public class Solution {
    static int doubleBeauty(int k, int[] a) {
        int n = a.length;
        Arrays.sort(a);
        int s = 0;

        for (int i = 0; i < k; i++) {
            s += a[n - 1 - i] * 2;
        }

        return s;
    }
}

File: Challenges/Coding World Cup/2023/Day 30 - NED vs AFG - Double Score Beauty.py
from typing import *


def doubleBeauty(k: int, a: List[int]) -> int:
    # write your code here
    n = len(a)
    a.sort()
    s = 0

    for i in range(k):
        s += a[n - 1 - i] * 2

    return s


